src/styles/_catalog.scss

```diff
@@ -1,22 +1,27 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  justify-content: center;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: 488px) {
```

Comment:
you can use sass @each or maybe function to do code smoller

Reply:
Agree, loop can be useful here. And usually we using mixins for typical breakpoints